### DISCUSSION TITLE
Number of seats 0. However, paying for 1 that's not occupied.

### DISCUSSION THREAD

**Discussion by KooroshKhad** [community/community#40334] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have paid for 2 seats there's currently 1 member in the Org and 0 seats remaining. There are no pending invites.

I've recently started using the Organisation feature from GitHub to get a feel for it. I invited the wrong person and once they accepted the seat I removed them. Now that I have, I'm paying for 2 and can't invite a second.

**Comment by ossanna16** [comment]
Hi there @KooroshKhad and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by DrkDevil** [comment]
This is happening to me as well.

**Comment by alexkoven** [comment]
Same here. It says I have two more available seats and it doesn't let me add anybody.

**Comment by kirby-jack** [comment]
Same, im paying for 1 and removed the one person, but now it's saying 0 available.

**Comment by shinybrightstar** [comment]
Hi @KooroshKhad! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Comment by jaewonmoon00** [comment]
same issue. Can anyone fix this? There is no ticket for this problem on support page
